perf/load/common.sh

```diff
@@ -52,6 +54,10 @@ function run_test() {
     kubectl -n "${ns}" delete -f "${YAML}" || true
     kubectl delete ns "${ns}"
   fi
+  if [[ "${VM_ENABLED}" == "true" ]]; then
+    VM_NAMESPACE="${ns}" VM_APP="${prefix}0-9"  ./bootstrap-vm.sh
```

Comment:
One more comment, are the 0-9 and 0-9-0 ones just randomly chosen from the graph?

Reply:
the last ones currently are -0-8 and 0-8-0, so I just picked the next number.

I kind of think we should redo the whole graph and make it more clear like label the TCP ones, and name these ones "VM"